openapi-v1.yaml

```diff
@@ -1391,6 +1391,12 @@ definitions:
       default_namespace_charged:
         description: Override the default namespace charged for actions when no namespace is specified
         type: string
+      default_region:
+        description: >
+          The default AWS region to use for notebooks and other operations. It must be a Region from
```

Comment:
Lets make the wording a little more generic to support other cloud providers too.

Do we need a second field for "default provider" or similar? I'm just thinking about how we do `region`.`vendor`.api.tiledb.com today?

cc: @snagles and @antalakas for throughts

Reply:
My opinion is that we can make the wording more generic (like remove AWS and avoid the "It must be ... "), but at the same time avoid adding a default provider. It will be easy to add it later and probably when we will the need for it we will rework fields in the config and take better code decisions too.

Reply:
What do you think of
```
The default region to use for notebooks and other operations. It must be a region supported by TileDB, one of: us-east-1.aws, us-west-2.aws, eu-west-2.aws, ap-southeast-1.aws
```

I would like to remove the regions enumerations but i can't find a TileDB cloud page that mentions them to put the URL here

Reply:
Wording is good, regions are listed here: https://docs.tiledb.com/cloud/concepts/tiledb-cloud-internals/architecture#orchestration .